### DISCUSSION TITLE
studio.grapesjs.com

### DISCUSSION THREAD

**Discussion by Gabbar-v7** [GrapesJS/grapesjs#5980] [General]
Online grapesjs builder tools at studio.grapesjs.com doesnt work.
Also another question if I build github.com/grapesjs/grapesjs locally will work like the studio.grapesjs.com online

Please atleast give the alpha version of the builder for free to download atleast i was building a project using it and now i am stuck half way.
![Screenshot_20240629-200253_Brave](https://github.com/GrapesJS/grapesjs/assets/147980619/da3cbd8e-13e7-4d68-a2c8-77251047abdb)
![Screenshot_20240629-200306_Brave](https://github.com/GrapesJS/grapesjs/assets/147980619/4892be3f-3cdd-431b-be2d-5c2ecc060584)

Below is a screen shot of the result at studio.grapesjs.com on both mobile and windows

**Comment by artf** [comment]
Thanks @Gabbar-v7 I didn't notice a misconfiguration on the DNS level, it was working on my side due to the PWA 🥲
Try to check now, it should be working

**Reply by mosh-tudor** [reply]
@artf 

Can you share your plans for studio.grapesjs.com? 

What will it be?
Online service (like webflow)?
Or an update for GrapesJs?
Or maybe a new library?

The lack of information is a bit demotivating in creating plugins :-(

**Reply by artf** [reply]
Here is the discussion: https://github.com/GrapesJS/studio/discussions/2#discussioncomment-9687573
Feel free to ask more questions.

**Reply by Gabbar-v7** [reply]
It is working now thank you.

**Reply by Gabbar-v7** [reply]
> Here is the discussion: [GrapesJS/studio#2 (comment)](https://github.com/GrapesJS/studio/discussions/2#discussioncomment-9687573) Feel free to ask more questions.

Can u atleast make the current studio.grapesjs.com builder free for everyone or atleast give the code or build version to use it offline.
Not now but atleast when u decide to remove it from online website

**Reply by artf** [reply]
> Can u atleast make the current studio.grapesjs.com builder free for everyone

@Gabbar-v7 it's already free for everyone 😅

> when u decide to remove it from online website

who is saying I'll remove it??? It's a standalone app, you can download it and use it even offline...

**Reply by Gabbar-v7** [reply]
> who is saying I'll remove it??? It's a standalone app, you can download it and use it even offline...

😀😀
Nice I love it.

But how to download it and use offline?

**Reply by mosh-tudor** [reply]
@Gabbar-v7 
<img width="508" alt="Screenshot 2024-07-03 at 18 19 17" src="https://github.com/GrapesJS/grapesjs/assets/668451/34855584-4b14-4c87-a10c-a653eb201f1a">

**Reply by Gabbar-v7** [reply]
> @Gabbar-v7 <img alt="Screenshot 2024-07-03 at 18 19 17" width="508" src="https://private-user-images.githubusercontent.com/668451/345519363-34855584-4b14-4c87-a10c-a653eb201f1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAwNjM3MDUsIm5iZiI6MTcyMDA2MzQwNSwicGF0aCI6Ii82Njg0NTEvMzQ1NTE5MzYzLTM0ODU1NTg0LTRiMTQtNGM4Ny1hMTBjLWE2NTNlYjIwMWYxYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwNFQwMzIzMjVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01OTAyOTJjMWFmN2FlNzNjNDlhZDdkMmM5ZWY4ZDg2MDc2NzI1OWEwNTI5MzQ2NTRmOWIxYzAxOWU1ZmEzOWVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.o74qV9c8zx8RzEH7mZoRUhQqKCV6IPNsoAfVvpEeaD8">

Got it bro 😁😁
